flink/v1.17/flink/src/main/java/org/apache/iceberg/flink/sink/IcebergFilesCommitter.java

```diff
@@ -131,13 +134,15 @@ class IcebergFilesCommitter extends AbstractStreamOperator<Void>
       Map<String, String> snapshotProperties,
       Integer workerPoolSize,
       String branch,
-      PartitionSpec spec) {
+      PartitionSpec spec,
+      long minReloadIntervalMs) {
     this.tableLoader = tableLoader;
     this.replacePartitions = replacePartitions;
     this.snapshotProperties = snapshotProperties;
     this.workerPoolSize = workerPoolSize;
     this.branch = branch;
     this.spec = spec;
+    this.minReloadIntervalMs = minReloadIntervalMs;
```

Comment:
My comment might be lost somewhere, but I think it would be better to receive the TableSupplier here, instead of creating a new "incidentally" same supplier than we do in the FlinkSink.

Also seeing the TableLoader/TableSupplier in one place mad me think how do we separate the functionality of the 2. Do we really need 2 objects providing/reloading tables?

Reply:
Oh.. I found my previous comment